recipes/mecab-ko-dic/run_test.bat

```diff
@@ -0,0 +1,4 @@
+mecab %RECIPE_DIR%\test.txt
```

Comment:
Following on from your [question on gitter](https://gitter.im/conda-forge/conda-forge.github.io?at=601462ee063b6c68d5271eb3): I think this is worth opening an issue against conda-build. Better yet, the PR to fix it looks like it might be straight forward. Just modify [these lines](https://github.com/conda/conda-build/blob/8f1b3517fd0c816ec78b6dadf4a912b849ecd58a/conda_build/build.py#L2412-L2414) like so:

```diff
-        with open(f) as _f:
+        with open(f, "rb") as _f:
             contents = _f.read()
-        if ("SRC_DIR" in contents and 'source_files' not in metadata.get_section('test') and
+        if (b"SRC_DIR" in contents and 'source_files' not in metadata.get_section('test') and
```